src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,565 @@
+
+# (c) Secure Identity Alliance
+
+openapi: 3.0.0
+info:
+  description: OSIA CMS Interface
+  version: 1.0.0
+  title: OSIA Credential Management System Interface
+  license:
+    name: SIA
+    url: "https://raw.githubusercontent.com/SecureIdentityAlliance/osia/master/LICENSE"
+tags:
+  - name: Credential Request
+servers:
+  - url: http://cms.com/
+  - url: https://cms.com/
+paths:
+
+  /v1/credentialRequest/person/{personId}:
+    post:
+      tags:
+        - Credential Request
+      summary: Create a request for a credential
+      operationId: createCredentialRequest
+      parameters:
+        - name: personId
```

Comment:
I think we need to provide more information about the person than just the personId.